src/components/Profile/ProfileImage/ProfileImage.css

```
@@ -0,0 +1,8 @@
+.profile__header__image img {
+  height: 150px;
+  overflow: hidden;
+}
+
+.profile__header__image__circular img {
+  border-radius: 50%;

```

```suggestion
  border-radius: 9999px;
```

I discovered recently this cool trick (see [here](https://github.com/tailwindlabs/tailwindcss/discussions/3373))